files/jenkins.json

```
@@ -1,33 +1,34 @@
 {
     "plugins": [
         { "name": "apache-httpcomponents-client-4-api", "version": "4.5.14-269.vfa_2321039a_83" },
-        { "name": "asm-api", "version": "9.8-135.vb_2239d08ee90" },
-        { "name": "bouncycastle-api", "version": "2.30.1.80-256.vf98926042a_9b_" },
-        { "name": "branch-api", "version": "2.1217.v43d8b_b_d8b_2c7" },
-        { "name": "caffeine-api", "version": "3.2.0-166.v72a_6d74b_870f" },
-        { "name": "cloudbees-folder", "version": "6.1012.v79a_86a_1ea_c1f" },
-        { "name": "display-url-api", "version": "2.209.v582ed814ff2f" },
-        { "name": "durable-task", "version": "587.v84b_877235b_45" },
-        { "name": "instance-identity", "version": "201.vd2a_b_5a_468a_a_6" },
-        { "name": "ionicons-api", "version": "82.v0597178874e1" },
+        { "name": "asm-api", "version": "9.9-185.va_6c6b_3348b_c3" },
+        { "name": "bouncycastle-api", "version": "2.30.1.82-277.v70ca_0b_877184" },
+        { "name": "branch-api", "version": "2.1259.v45c101731c76" },
+        { "name": "caffeine-api", "version": "3.2.3-194.v31a_b_f7a_b_5a_81" },
+        { "name": "cloudbees-folder", "version": "6.1073.va_7888eb_dd514" },
+        { "name": "commons-lang3-api", "version": "3.19.0-104.v12125f33a_255" },
+        { "name": "display-url-api", "version": "2.217.va_6b_de84cc74b_" },
+        { "name": "durable-task", "version": "635.v3733cef34b_5e" },
+        { "name": "instance-identity", "version": "203.v15e81a_1b_7a_38" },
+        { "name": "ionicons-api", "version": "94.vcc3065403257" },
         { "name": "jakarta-activation-api", "version": "2.1.3-2" },
-        { "name": "jakarta-mail-api", "version": "2.1.3-2" },
+        { "name": "jakarta-mail-api", "version": "2.1.3-3" },
         { "name": "javax-activation-api", "version": "1.2.0-8" },
         { "name": "javax-mail-api", "version": "1.6.2-11" },
-        { "name": "mailer", "version": "489.vd4b_25144138f" },
-        { "name": "pipeline-groovy-lib", "version": "752.vdddedf804e72" },
-        { "name": "scm-api", "version": "704.v3ce5c542825a_" },
-        { "name": "script-security", "version": "1373.vb_b_4a_a_c26fa_00" },
-        { "name": "structs", "version": "343.vdcf37b_a_c81d5" },
+        { "name": "mailer", "version": "522.va_995fa_cfb_8b_d" },
+        { "name": "pipeline-groovy-lib", "version": "776.vfee5327b_b_a_5b_" },
+        { "name": "scm-api", "version": "712.v8846fdd68c88" },
+        { "name": "script-security", "version": "1385.v7d2d9ec4d909" },
+        { "name": "structs", "version": "353.v261ea_40a_80fb_" },
```

If we're having to make a change anyway, might as well catch the two updates that have been brought in since Friday:

```suggestion
        { "name": "structs", "version": "362.va_b_695ef4fdf9" },
```